app/assets/stylesheets/getting-started.scss

```diff
@@ -1,7 +1,11 @@
 #hello-there {
-  .avatar {
-    width: 200px;
-    height: 200px;
```

Comment:
> I removed it because it didn't affect the new avatar.

It does affect the new avatar:

![new user avatar](https://cloud.githubusercontent.com/assets/3798614/8261933/c09718a0-16ce-11e5-9fe8-85feaed437ab.png)